tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py35, py36
+envlist = py27, py35, py36, py37
```

Comment:
Can we drop one of py35 or py36 versions?

Reply:
37 is still too new, 35 and 36 is still widely used. we can revisit this when we are running into any issue with older version of pythons. Right now most of the difference doesn't apply to our code base.